version/version.go

```diff
@@ -12,6 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package main
+package version
 
-const Version = "0.5.3+git"
+var Version = "0.5.3+was-not-built-properly"
```

Comment:
Yeah but then we still need the bump revision dance. I like that `git describe` magic as it cleans it up.